pages/registries.py

```diff
@@ -1,16 +1,34 @@
 import settings
 
+from urllib.parse import urljoin
 from selenium.webdriver.common.by import By
 
-from base.locators import Locator, ComponentLocator
+from base.locators import Locator, ComponentLocator, GroupLocator
 from components.navbars import RegistriesNavbar
-from pages.base import OSFBasePage
+from pages.base import OSFBasePage, GuidBasePage
 
 
-class RegistriesPage(OSFBasePage):
+class BaseRegistriesPage(OSFBasePage):
+
+    # Components
+    navbar = ComponentLocator(RegistriesNavbar)
+
+class RegistriesLandingPage(BaseRegistriesPage):
     url = settings.OSF_HOME + '/registries'
 
     identity = Locator(By.CSS_SELECTOR, '._RegistriesHeader_3zbd8x', settings.LONG_TIMEOUT)
+    search_button = Locator(By.CSS_SELECTOR, '[data-test-search-button]')
 
-    # Components
-    navbar = ComponentLocator(RegistriesNavbar)
+class RegistriesDiscoverPage(BaseRegistriesPage):
+    url = settings.OSF_HOME + '/registries/discover'
+
+    identity = Locator(By.CSS_SELECTOR, '[data-test-share-logo]')
+    loading_indicator = Locator(By.CSS_SELECTOR, '.ball-scale')
+
+    # Group Locators
+    search_results = GroupLocator(By.CSS_SELECTOR, '._SearchResult_10ty34')
+
+class RegistrationDetailPage(GuidBasePage):
+    url_base = urljoin(settings.OSF_HOME, '{guid}')
+    identity = Locator(By.CSS_SELECTOR, '[data-test-result-title-id]')
```

Comment:
This identity doesn't seem right. That looks like a selector from an embosfed page. The registration detail page isn't emberized.